.github/workflows/publish.yml

```
@@ -57,7 +57,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:  # using `matrix` to define a constant
-        package: ['julia==0.5.8.dev']
+        package: ['julia==0.6.0.dev']
```

```suggestion
        package: ['julia==0.6.0']
```

(So we can create a new release)

---

Unless we release 0.5.8 on the current master branch; then we could leave this as 0.6.0.dev.

---

For 0.5.8, I think we should create a branch before we dropped Python 2.7 support and proceed that way.